src/sessions/softphone-session-handler.ts

```diff
@@ -568,15 +559,15 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
               endedSession,
               conversationId,
               session
-            }, { skipServer: true });
```

Comment:
These would be useful to send up to supportability but I'm not sure if they'll have PII?

Reply:
by default, `debug` level logs don't go to sumo. I think we should log some sort of ended event, however we don't want to log full session objects (which is what is happening here). Let's leave the `skipServer` on this debug level log and add an `info` level log that logs the sessionIds, conversationId, and sessionType.